src/hooks/useMatchingEvmNetwork.ts

```diff
@@ -16,11 +16,10 @@ export const useMatchingEvmNetwork = ({
   switchAutomatically?: boolean;
   onError?: (error: Error) => void;
 }) => {
-  const { chain } = useNetwork();
+  const { chain } = useAccount();
```

Comment:
Wagmi and privy has mostly syntax changes here, eg switchChain instead of switchNetwork